### PR TITLE
Highlight guidance section

### LINKED ISSUE #5781: Exemplar solution in "guidance" in mentoring UI lacks syntax highlighting
Screenshot 2021-09-05 at 12-30-44 Exercism

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/exercism/issues/5781.

### CODE REVIEW

File: app/javascript/components/mentoring/session/MentorNotes.tsx [hunk 1]
```diff
@@ -24,12 +23,10 @@ export const MentorNotes = ({
     )
   }
 
-  const ref = useHighlighting<HTMLDivElement>()
   return (
     <React.Fragment>
       <div
         className="c-textual-content --small"
-        ref={ref}
         dangerouslySetInnerHTML={{ __html: notes }}
```
Comment (iHiD) [inline app/javascript/components/mentoring/session/MentorNotes.tsx:30]: @kntsoriano Why have you removed highlighting from the notes?

**Comment by iHiD** [comment]
This looks good. Just one question before I merge.

**Comment by kntsoriano** [comment]
@iHiD The mentor notes are still highlighted. I decided to apply the highlighting to the whole block. The mentor notes are within it. Do you want me to retain the highlighting as we might reuse the mentor notes somewhere else?

**Comment by iHiD** [comment]
OK, cool. It doesn't matter that the "how you solved it" block is already highlighted within this?

File: app/javascript/components/mentoring/session/Guidance.tsx [hunk 2]
```diff
@@ -108,8 +111,9 @@ export const Guidance = ({
               </p>
               <pre className="overflow-auto">
                 <code
+                  className={language}
                   dangerouslySetInnerHTML={{ __html: exemplarSolution }}
```
Comment (joshgoebel) [inline app/javascript/components/mentoring/session/Guidance.tsx:115]: Is `exemplarSolution` here already properly HTML escaped?

Comment (iHiD) [reply app/javascript/components/mentoring/session/Guidance.tsx:115]: Probably not

Comment (iHiD) [reply app/javascript/components/mentoring/session/Guidance.tsx:115]: @joshgoebel Does it need to be on a `<code>` elem though?

Comment (joshgoebel) [reply app/javascript/components/mentoring/session/Guidance.tsx:115]: Yes, always, HTML inside `<code>` is still just HTML (if not escaped):

```html
<code>
<script>
destroyWebsite()
</script>
</code>
```

Not good.

Comment (iHiD) [reply app/javascript/components/mentoring/session/Guidance.tsx:115]: https://github.com/exercism/website/pull/1886

**Pull request merged**